libs/sdk-backend-tiger/src/backend/organization/settings.ts

```diff
@@ -18,6 +18,14 @@ export class OrganizationSettingsService
         return this.setSetting("WHITE_LABELING", whiteLabeling);
     }
 
+    public async setTimeZone(timezoneId: string): Promise<void> {
+        return this.setSetting("TIMEZONE", { value: timezoneId });
+    }
+
+    public async setDateFormat(formatId: string): Promise<void> {
+        return this.setSetting("FORMAT_LOCALE", { value: formatId });
```

Comment:
Just a confirmation, is it `"FORMAT_LOCALE"` instead of something like `"DATE_FORMAT"`?